src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color, "circle");
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    protected String getAdditionalInfo() {
```

Comment:
The method name 'getAdditionalInfo' could be more specific to the context of the Circle class. Consider renaming it to reflect the information it provides, such as 'getRadiusInfo'.